actsims/simTools.py

```diff
@@ -1,6 +1,7 @@
 
 
-import numpy as np, flipper.flipperDict as flipperDict, pickle
+import numpy as np, pickle
+from . import flipperDict
```

Comment:
Where is flipperDict coming from? Is it part of /actsims directory now?

Reply:
ah I see it is not in there. I will add it.

Reply:
fixed.